styling/Phoenix_files/_hero.scss

```diff
@@ -84,3 +85,25 @@ body, h1, h2, h3, h4, h5, h6 {
   background-image: url("../Phoenix_files/phoenix-logo-white.png") !important;
   background-size: 250px 34px;
 }
+
+#subheader-columns {
+  margin: 50px 30px 0; }
+
+.subheader h4 {
+  color: #000;
+  font-weight: $light-font;
+  font-size: 18px;
+  padding-bottom: 14px;
+  text-transform: uppercase; }
+
+.subheader p {
+  font-weight: $light-font;
+  color: #333;
```

Comment:
The problem is that it is not really the color of the body... For example, the auto generated stuff is slightly lighter. Maybe $subheader-title-color and $subheader-color? For now I don't expect to re-use it anywhere... even though, is it a good practice to make them variables?